.travis.yml

```diff
@@ -1,8 +1,11 @@
 
 language: node_js
+os:
+  - linux
+  - osx
 node_js:
-  - "5"
+  - "8"
```

Comment:
Is `8` supported in travis yet? - https://docs.travis-ci.com/user/languages/javascript-with-nodejs/